README.md

```diff
@@ -1,2 +1,216 @@
+    * range: extracted from sitename
+    * column: extracted from sitename
+    * cultivar: sorghum variety
+    * date_of_flag_leaf_emergence: year-month-day format
+    * days_to_flag_leaf_emergence: days from planting to flag leaf emergence
+    * gdd_to_flag_leaf_emergence: cumulative growing degree days to flag leaf emergence
+
+* **days_gdd_to_flowering_season_4_timestamp.csv**
+    #### Columns
+    * sitename: location-based identifier, MAC Field Scanner Season 4 Range x Column y
+    * range: extracted from sitename
+    * column: extracted from sitename
+    * cultivar: sorghum variety
+    * date_of_flowering: year-month-day format
+    * days_to_flowering: days from planting to flowering
+    * gdd_to_flowering: cumulative growing degree days to flowering
+
+### II. MAC Season 6
+#### A. Environmental 
+* **weather_station_season_6_timestamp.csv**
+    #### Columns - information from [Weather Station](https://cals.arizona.edu/azmet/raw2003.htm)
+    * date: year-month-day format
+    * day of year - information from [NOAA](https://www.esrl.noaa.gov/gmd/grad/neubrew/Calendar.jsp?view=DOY&year=2017&col=4)
+    * air_temp_max: degrees, Celsius
+    * air_temp_min: degrees, Celsius
+    * air_temp_mean: degrees, Celsius
+    * rh_max: relative humidity, percentage
+    * rh_min: relative humidity, percentage
+    * rh_mean: relative humidity, percentage
+    * precip_total: precipitation total, mm
+    * wind_speed_mean: m/s, meters per second
+    * wind_vec_mag: wind vector magnitude in m/s, meters per second, calculation using direction and speed
+    * wind_vec_dir: wind vector direction, degrees
+        * wind blowing from the north = 0&deg;/360&deg;
+        * wind blowing from the east = 90&deg;
+        * wind blowing from the south = 180&deg;
+        * wind blowing from the west = 270&deg;
+    * wind_dir_std: wind direction standard deviation, degrees
+    * max_wind_speed: m/s, meters per second
+    * gdd: cumulative growing degree days, using a base/threshold temperature of 10&deg;C 
+    * cum_precip: cumulative precipitation, mm
+#### B. Phenotypes
+* **tall_format_traits_season_6_timestamp.csv**
+    #### Columns
+    * date: year-month-day format
+    * sitename: location-based identifier, MAC Field Scanner Season 4 Range x Column y
+    * range: extracted from sitename
+    * column: extracted from sitename
+    * lat: latitude
+    * lon: longitude
+    * cultivar: sorghum variety
+    * trait: value
+        * canopy_height: cm
+        * aboveground_dry_biomass: kg/ha, kilogram per hectare, whole aboveground biomass at harvest 
+
+## Sharing and Access Information
+TERRA-REF Data Use [Policy](https://docs.terraref.org/user-manual/data-use-policy)
+
+## Methodological Information
+
+### I. Data Collected By: 
+### II. Data Collection Methods:
+* [Manual Field Data Protocols](https://docs.terraref.org/protocols/manual-field-data-protocols) from TERRA-REF docs
+
+### III. Data Curation: 
+
```

Comment:
all of the code used should be in this repository, so that someone can complete the analysis within a single jupyter notebook, script, or makefile (in R, drake is popular)

If the code came from these other sources, then they can be referenced, but you should not assume that these links will be stable, and surely there were changes from the precise code that you used. These references could go at the end under 'further reading' or such, and links can go inline in the code or notebook as is useful.